subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -418,6 +418,10 @@ enum pdu_data_llctrl_type {
 	PDU_DATA_LLCTRL_TYPE_PHY_RSP = 0x17,
 	PDU_DATA_LLCTRL_TYPE_PHY_UPD_IND = 0x18,
 	PDU_DATA_LLCTRL_TYPE_MIN_USED_CHAN_IND = 0x19,
+	PDU_DATA_LLCTRL_TYPE_CIS_REQ = 0x1F,
```

Comment:
You have to fill in the missing ones for the LUT to work:
PDU_DATA_LLCTRL_TYPE_CTE_REQ = 0x1A
PDU_DATA_LLCTRL_TYPE_CTE_RSP = 0x1B
PDU_DATA_LLCTRL_TYPE_PERIODIC_SYNC_IND = 0x1C
PDU_DATA_LLCTRL_TYPE_CLOCK_ACCURACY_REQ = 0x1D
PDU_DATA_LLCTRL_TYPE_CLOCK_ACCURACY_RSP = 0x1E